### ISSUE TITLE
Purchases: Refund Survey: Check boxes can't be selected using keyboard-only 

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/wp-calypso#6512]
While testing #4838: It's not possible to select items in the survey with the <kbd>Enter</kbd> key.

#### Steps to reproduce
1. Add any upgrade of your choosing to a WordPress.com site.
2. Navigate to **/purchases** and select your newly added upgrade.
3. Select the option to remove the upgrade.
4. The refund survey will appear in a new window with two questions: **Please tell us why you are canceling** and **Where is your next adventure taking you?**
5. Tab through each of the options under each question using the <kbd> ↹</kbd> tab key.
6. Attempt to select an option using the <kbd>Enter</kbd> key.

#### What I expected

I expected for a box to be "checked" after <kbd> ↹</kbd> tabbing to it and then pressing the <kbd>Enter</kbd> key. 

#### What happened instead

Pressing the <kbd>Enter</kbd> key on a check box has no effect. This would make it hard for those navigating the web solely on their keyboard to progress through to the next stage of the survey. 

#### Browser / OS version

Chrome on Mac OS 10.11.5.

#### Screenshot / Video

<img width="565" alt="screen shot 2016-07-03 at 16 53 25" src="https://cloud.githubusercontent.com/assets/2998162/16546328/e86e7ae4-413e-11e6-809c-e9e4c04f9043.png">

#### Context / Source

 #empathy #perspective #manual-testing

**Comment by jeremeylduvall** [comment]
This is currently working with the spacebar on my side in Chrome. I think the spacebar is used on radio/checkboxes whereas the enter key would be on submit. Can you confirm spacebar is working for you as well?

**Comment by SiobhyB** [comment]
@jeremeylduvall: Ah, I didn't realise that [the spacebar is used to select radio/checkboxes](http://webaim.org/techniques/keyboard/#testing)! I had made the assumption that the enter key was always used. 

I tested and can confirm that the spacebar is working for me. Thank you for testing and clarifying that that's the correct behaviour.

I'm closing this issue and will be mindful about making assumptions in the future.
